### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by jnm2**
Explanations in commit messages. The final commit moves everything around.

IconUrl deprecation: https://docs.microsoft.com/en-us/nuget/reference/nuspec#iconurl
LicenseUrl deprecation: https://docs.microsoft.com/en-us/nuget/reference/nuspec#licenseurl

Authors recommended: https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata

### CODE REVIEW

File: src/Shouldly/Shouldly.csproj [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-    <PackageLicenseUrl>https://github.com/shouldly/shouldly/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <LangVersion>latest</LangVersion>
+    <Optimize>false</Optimize>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <AssemblyOriginatorKeyFile>sn.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  
+    <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
+    <Authors>Jake Ginnivan, Joseph Woodward</Authors>
```
Comment (jnm2) [inline src/Shouldly/Shouldly.csproj]: Is this right?

File: src/Shouldly/Shouldly.csproj [hunk 2]
```diff
@@ -41,4 +42,7 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <DefineConstants>$(DefineConstants);ShouldMatchApproved;dNewReflection;ExpressionTrees;StackTrace;Serializable;CallContextPolyfill</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>
+    <None Include="..\..\assets\logo_128x128.png" Pack="true" PackagePath="assets" />
+  </ItemGroup>
```
Comment (jnm2) [inline src/Shouldly/Shouldly.csproj]: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file

**Comment by jnm2** [comment]
@JosephWoodward

File: src/Shouldly/Shouldly.csproj (hunk 1 above)
Comment (JosephWoodward) [reply src/Shouldly/Shouldly.csproj]: Yes, that's right. Though I feel it should be everyone who has contributed in there personally. It always feels odd distilling a large OSS contribution such as this down to two names.

**Comment by JosephWoodward** [comment]
This looks great, thanks. I'm just finishing off work on some other libraries at the moment then going to dedicate a good amount of time to getting some of these pull requests reviewed, merged and released (and the build a bit more stable!), but I'll merge this one in the meantime as it's small.

**Comment by jnm2** [comment]
Yeah, NRTs would be a good one to merge last.
 :D

**Pull request merged**